test/integration/Bravo01.t.sol

```diff
@@ -2,13 +2,16 @@ pragma solidity ^0.8.0;
 
 import {Vault} from "@examples/Vault.sol";
 import {MockToken} from "@examples/MockToken.sol";
+import {BRAVO_01} from "@examples/governor-bravo/BRAVO_01.sol";
+
 import {GovernorBravoPostProposalCheck} from "@test/GovernorBravoPostProposalCheck.sol";
-import "@forge-std/Test.sol";
 
-// @dev This test contract extends GovernorBravoPostProposalCheck, granting it
-// the ability to interact with state modifications effected by proposals
-// and to work with newly deployed contracts, if applicable.
-contract GovernorBravoProposalTest is GovernorBravoPostProposalCheck {
+contract Bravo01IntegrationTest is GovernorBravoPostProposalCheck {
+    function setUp() public override {
+        proposal = new BRAVO_01();
```

Comment:
`proposal` variable is declared on GovernorBravoPostProposalCheck but we initiate on the test file so that each test file can initiate it own proposal.